Meta/Keywords/scripting_library/UI/plots/displayed_elements.py

```diff
@@ -266,6 +270,7 @@ async def _get_cached_values_to_display(self, cached_value_metadata, x_shift, st
         cache_displayed_value = plotted_displayed_value = cached_value_metadata["cache_value"]
         kind = cached_value_metadata["kind"]
         mode = cached_value_metadata["mode"]
+        line_shape = cached_value_metadata["line_shape"]
```

Comment:
this requires the plot keyword to set "line_shape"

Reply:
thats already added
![image](https://user-images.githubusercontent.com/32987678/221402563-99c284b6-2133-45ec-b8c4-81978bb9a484.png)